### DISCUSSION TITLE
Wire buffer size: Code vs Docs?

### DISCUSSION THREAD

**Discussion by microfoundry** [earlephilhower/arduino-pico#1988] [General]
Docs state: "Master transmissions are buffered (up to 128 bytes) ...."
Wire.h has: #define WIRE_BUFFER_SIZE 256
Is it actually 256 bytes? Or am I missing something?

And the onIRQ() RX section utilizes the same _buff, so I assume it could receive up to 256 bytes?

-Terry

**Comment by earlephilhower** [comment]
Yeah, that looks like a change that didn't make it into the docs.  Want to do a PR to fix the docs?

**Reply by microfoundry** [reply]
PR submitted - thank you for the clarification.

-Terry
